elastic_django/tests/test_reservation_unit_query_builder.py

```diff
@@ -0,0 +1,54 @@
+from assertpy import assert_that
+from django.test import TestCase
+
+from elastic_django.reservation_units.query_builder import (
+    ReservationUnitQueryBuilderMixin,
+)
+
+
+class ReservationUnitQueryBuilderTestCase(TestCase):
+    def test_parse_search_word_when_one_search_word(self):
+        query_str = ReservationUnitQueryBuilderMixin.build_elastic_query_str("search")
+
+        assert_that(query_str).is_equal_to("(*search*)")
+
+    def test_parse_search_word_when_two_search_word(self):
+        query_str = ReservationUnitQueryBuilderMixin.build_elastic_query_str(
+            "search this"
+        )
+
+        assert_that(query_str).is_equal_to("((*search) AND (this*))")
```

Comment:
Does this also match to text when there are words in between the two key words? For example, does `((*search) AND (this*))` match to `search foobar this`? If yes, is this the intended behaviour or should it be something like `(*search AND this*)`?